src/components/SearchResults/SearchResults.css

```diff
@@ -1,3 +1,21 @@
 .results article:nth-of-type(even) {
   background: lightgray;
 }
+
+.results {
+  width: 100%;
+}
+
+/*
+FIX DEM A TAGS
+*/
+.user-container {
+  display: flex;
+  justify-content: space-evenly;
+  width: 100%;
+}
+
+
+.user-container a article *{
```

Comment:
I wonder why this isn't working, we'll have to dig deeper on this styling eventually